### ISSUE TITLE
Clicking on published worlds also selects the shuffle tab

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#2512]
### Describe the issue.

Clicking on published worlds also selects the shuffle tab

### To Reproduce

Click on the published worlds tab and you'll notice that the shuffle tab is also highlighted

### Expected behavior

It should only select the published world's tab

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/9aa95759-567d-4f5a-942e-67c3c35c3231)

### Resonite Version Number

Beta 2024.7.8.1285

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[EPICEASTON197 - 2024.7.8.1285 - 2024-07-09 07_07_25.log](https://github.com/user-attachments/files/16142436/EPICEASTON197.-.2024.7.8.1285.-.2024-07-09.07_07_25.log)

### Reporters

@epicEaston197

**Comment by epicEaston197** [comment]
Ohh never mind this happens with the current build xD
Guess I just never noticed

**Comment by Zyro1331** [comment]
I was able to replicate the issue with Beta 2024.7.4.1347 (Stable Release)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/49619936/16e80217-a88a-43f0-aa2a-a43869372928)

Clicking on either Shuffle or Published Worlds gives the exact same results interestingly. I am not sure if this was caused by the back and forth of going between both builds messing up the facet, as I still use the Published Worlds category fairly often and never noticed this behavior until recently.

**Comment by epicEaston197** [comment]
I can confirm logging out fixes the issue

**Comment by Zyro1331** [comment]
There's a chance this could be related to https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2509? 
My theory is that Resonite might be trying to automatically upgrade that specific facet to an incorrect version or somehow mangling it after users opt-into the prerelease build.

**Comment by shiftyscales** [comment]
I don't believe so, @Zyro1331. My personal account hasn't touched this pre-release build yet as far as I recall, and I also see this issue on `Beta 2024.7.4.1347`.

So I don't think it's directly related to the pre-release itself. It's possible a previous upgrade to the facet caused an issue- but the issue doesn't seem to present itself on freshly generated copies of the facet.

As a sanity check, you could reset your dash see if the issue occurs, then go to the prerelease build and see if it occurs there. @Zyro1331 @epicEaston197.

**Comment by Zyro1331** [comment]
I performed said sanity check by running a dash reset on the stable build, switching to the prerelease branch, and testing to see if the behavior appears again. Both builds did not have this issue after the dash was reset.

**Comment by Nytra** [comment]
It has been like this since the WorldsDataFeed was added and the WorldListManager was renamed to LegacyWorldListManager, which caused some dynamic variables to become MissingComponent in the worlds facet

**Comment by shiftyscales** [comment]
Another instance of this has come up in issue #2536 with a slightly different result - highlighting active sessions and contacts sessions together.

> ### Describe the bug?
> 
> When selecting the tab "Contacts sessions", it will also select the "Active sessions" tab in the world browser and vice versa.
> 
> ### To Reproduce
> 
> 1. Open Resonite
> 2. Go to the worlds tab
> 3. Click on "Contacts sessions"
> 4. Observe the active sessions are also selected
> 
> ### Expected behavior
> 
> Either tab should be selected and not the two of them.
> 
> ### Screenshots
> 
> ![RuIo2UKulH](https://github.com/user-attachments/assets/43c4634b-92ed-4438-95cc-623957922105)
> 
> 
> ### Resonite Version Number
> 
> Beta 2024.7.11.1293
> 
> ### What Platforms does this occur on?
> 
> Windows
> 
> ### What headset if any do you use?
> 
> Desktop, Quest Pro
> 
> ### Log Files
> 
> [TETHYS - 2024.7.11.1293 - 2024-07-12 01_12_41.log](https://github.com/user-attachments/files/16184167/TETHYS.-.2024.7.11.1293.-.2024-07-12.01_12_41.log)
> 
> 
> ### Additional Context
> 
> Reporting this as a bug instead of content issue as the issue wasn't present with the previous stable version (2024.7.10.1399).
> 
> ### Reporters
> 
> @jae1911

**Comment by shiftyscales** [comment]
Another case of this came up in #3721 - but since newly made copies of the facet seem to work as intended- this suggests the facet may just need an internal version bump so that it replaces the existing broken version for remaining users that are still affected (if any).

Not something that needs a significant priority however since it is something that can be worked around, and existing cases should become fewer and further between.

**Comment by Krzeszny** [comment]
Similar issue here for months, Active also selects Contacts, and Active also shows empty headlesses. I'll reset my dash.

**Comment by epicEaston197** [comment]
> Similar issue here for months, Active also selects Contacts, and Active also shows empty headlesses. I'll reset my dash. I don't want to make another issue for that because as specific as it sounds, it's definitely the same one.

Just going to ua edit mode and replace the respective facets that should fix it you don't need a full reset

**Comment by Krzeszny** [comment]
> > Similar issue here for months, Active also selects Contacts, and Active also shows empty headlesses. I'll reset my dash. I don't want to make another issue for that because as specific as it sounds, it's definitely the same one.
> 
> Just go in UI edit mode and replace the problematic facets that should fix it you don't need a full reset

I did it. I had to ask someone for theirs. I couldn't find the worlds facets in Resonite Essentials.

**Comment by epicEaston197** [comment]
> I did it. I had to ask someone for theirs. I couldn't find the worlds facets in Resonite Essentials.

You could have also Created an empty object And attached this component `Radiant UI->Facets->WorldFilterFacetPreset`
This will automatically create the facet according to default settings

![Image](https://github.com/user-attachments/assets/3204b17c-54fa-428c-b610-c4fa78a9df09)

**Comment by shiftyscales** [comment]
Frooxius recently bumped the facet for MMC25 so this should be resolved- if for some reason it is not, please re-open or make a new issue.
